packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -95,6 +95,29 @@ export const bm25s = (model: ModelData): string[] => [
 retriever = BM25HF.load_from_hub("${model.id}")`,
 ];
 
+export const cxr_foundation = (model: ModelData): string[] => [
+	`!git clone https://github.com/Google-Health/cxr-foundation.git
+import tensorflow as tf, sys, requests
+sys.path.append('cxr-foundation/python/')
+
+# Install dependencies
+major_version = tf.__version__.rsplit(".", 1)[0]
+!pip install tensorflow-text=={major_version} pypng && pip install --no-deps pydicom hcls_imaging_ml_toolkit retrying
```

Comment:
Still a bit sad to have these installation steps in the "run this model" snippet. Any way we can link to some external documentation instead of this snippet _is_ the only official snippet we can find on the internet?